src/builder.rs

```diff
@@ -256,24 +324,125 @@ impl ActionInfo {
     }
 }
 
-/// A builder that constructs a [`Bundle`] from a set of notes to be spent, and recipients
+/// Builds a bundle containing the given spent notes and outputs.
+///
+/// The returned bundle will have no proof or signatures; these can be applied with
+/// [`Bundle::create_proof`] and [`Bundle::apply_signatures`] respectively.
+pub fn bundle<V: TryFrom<i64>>(
```

Comment:
For reviewability purposes (with whitespace changes hidden), please extract this to just below the `impl Bundle`.